zerver/tests/test_message_flags.py

```diff
@@ -1908,9 +1908,7 @@ def test_unsubscribed_stream_messages_unread(self) -> None:
             )
         self.assert_json_success(result)
         event = events[0]["event"]
-        self.assertEqual(
-            event["messages"], subscribed_stream_message_ids + unsubscribed_stream_message_ids
-        )
+        self.assertEqual(event["messages"], subscribed_stream_message_ids)
```

Comment:
Do any of the comments in this test need to be updated to reflect this change?

Reply:
I think the answer is no, since the only thing that changed here is the event no longer uselessly has these extra messages.

Reply:
\*shrug\* The above comment `The messages which don't have UserMessage rows will be ignored` is now more true than it used to be.